src/Featurizers/StandardScaleWrapperFeaturizer.h

```diff
@@ -0,0 +1,348 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#pragma once
+
+#include "Components/StatisticalMetricsEstimator.h"
+#include "Components/StandardDeviationEstimator.h"
+#include "Components/PipelineExecutionEstimatorImpl.h"
+#include "../Archive.h"
+
+namespace Microsoft {
+namespace Featurizer {
+namespace Featurizers {
+
+/////////////////////////////////////////////////////////////////////////
+///  \class         StandardScalerTransformer
+///  \brief         Scales values based on learned mean and standard deviation values.
+///
+template <typename InputT, typename TransformedT>
+class StandardScalerTransformer : public StandardTransformer<InputT, TransformedT> {
+public:
+    // ----------------------------------------------------------------------
+    // |
+    // |  Public Types
+    // |
+    // ----------------------------------------------------------------------
+    static_assert(Traits<TransformedT>::IsNullableType && Traits<TransformedT>::IsIntOrNumeric::value, "'TransformedT' must be a numeric type");
```

Comment:
Will TransformedT always be std::double? If not, it seems like it will always be a numeric type. This static_assert should assert that rather than if it is nullable. We should probably add a IsNumeric to Traits.h.

Reply:
Combining IsNullableType and IsIntOrNumeric, only numerical types are eligible. Does writing like this harm understanding it?

Reply:
After discussion, adding IsNumeric will not be included in this PR.